packages/bpk-component-section-list/src/BpkSectionListItem.module.scss

```diff
@@ -40,6 +40,7 @@
   }
 
   &__chevron {
+    display: contents;
```

Comment:
![Screenshot 2024-04-12 at 15 46 29](https://github.com/Skyscanner/backpack/assets/89925955/52cf9f18-229a-4ce6-b3d3-5553415d59d1)

I think previously the parent of the svg would have been that `button` & that has `display: flex` testing quickly locally I think that gives the same result. 

Tbh I'm not quite sure what `display: contents` does... never come across it before!

Reply:
Ah, here's a blurb on the use of `contents`:

> display: contents causes an element's children to appear as if they were direct children of the element's parent, ignoring the element itself. This can be useful when a wrapper element should be ignored when using CSS grid or similar layout techniques.

Reply:
Interesting, so it's like it ignores the `span` for the wrapping. Nice. Seems appropriate!

Reply:
Bizarrely, when I tried `flex`, it wasn't aligned as per the snapshot on `main` (zoomed mobile):
![Screenshot 2024-04-12 at 16 02 57](https://github.com/Skyscanner/backpack/assets/7976511/61722ab6-0468-4ae9-af9e-04c8dd4d6509)

When I changed it to `contents` it matches the original snapshot on `main` (zoomed mobile):
![Screenshot 2024-04-12 at 16 00 30](https://github.com/Skyscanner/backpack/assets/7976511/028547ea-f17a-4e63-9441-e01266de6b07)

To me, `flex` is the right one to use, but it doesn't match the snapshot on `main`.